src/main.cpp

```diff
@@ -0,0 +1,80 @@
+#include <iostream>
```

Comment:
I think that this file should be here: MASOS/1/program/exercise_1.cpp, not in src/main.cpp